lib/timer.py

```diff
@@ -20,12 +21,22 @@
 # OrderedDict is used to ensure each job is run sequentially in the order
 # listed as some jobs may depend on completion of the previous job.
 UACLIENT_JOBS = collections.OrderedDict(
-    {
-        "update_messaging": {
-            "job_func": update_apt_and_motd_messages,
-            "run_interval_seconds": 43200,  # 12 hours
-        }
-    }
+    [
+        (
+            "update_messaging",
+            {
+                "job_func": update_apt_and_motd_messages,
+                "run_interval_seconds": 43200,  # 12 hours
+            },
+        ),
+        (
+            "update_status",
+            {
+                "job_func": update_status,
+                "run_interval_seconds": 43200,  # 12 hours
```

Comment:
Let's make this only daily. This could potentially mean lots of repeated traffic to the contract server, and 1. customers won't like a lot of pings egressing and 2. contract server might not want a lot of repeated loads.
```suggestion
                "run_interval_seconds": 60*60*24,  # 24 hours
```